### PR TITLE
Add collaborator methods

### PR DESCRIPTION
**Pull request by Joonpark13**
Adding https://docs.github.com/en/rest/collaborators/collaborators#add-a-repository-collaborator and https://docs.github.com/en/rest/collaborators/collaborators#remove-a-repository-collaborator

### CODE REVIEW

File: src/main/java/com/spotify/github/v3/clients/RepositoryClient.java [hunk 1]
```diff
@@ -475,7 +487,7 @@ public CompletableFuture<Optional<CommitItem>> merge(
                   github
                       .json()
                       .fromJsonUnchecked(
-                          GitHubClient.responseBodyUnchecked(response), CommitItem.class);
+                          responseBodyUnchecked(response), CommitItem.class);
```
Comment (dziemba) [inline src/main/java/com/spotify/github/v3/clients/RepositoryClient.java:490]: Why is this changed?

**Review by dziemba — DISMISSED** [review]
Looks good 👍

File: src/main/java/com/spotify/github/v3/clients/RepositoryClient.java (hunk 1 above)
Comment (Joonpark13) [reply src/main/java/com/spotify/github/v3/clients/RepositoryClient.java]: Must've been IntelliJ, reverted.

**Review by dziemba — APPROVED** [review]

**Comment by dziemba** [comment]
This is now released as version `0.1.33`
